<ui version="4.0" >
 <class>ICatSearch</class>
 <widget class="QWidget" name="ICatSearch" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>ICatSearch</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="3" column="0" >
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QPushButton" name="helpButton" >
       <property name="text" >
        <string>Help</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>527</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton" >
       <property name="text" >
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <widget class="QTreeWidget" name="searchtreeWidget" >
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>131</height>
      </size>
     </property>
     <property name="toolTip" >
      <string>ICat Search Box</string>
     </property>
     <property name="statusTip" >
      <string/>
     </property>
     <property name="styleSheet" >
      <string notr="true" >background-color: rgb(216, 225, 255);</string>
     </property>
     <column>
      <property name="text" >
       <string>                                                                                                                    Search Box</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="searchlabel" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QTableWidget" name="searchtableWidget" >
     <property name="toolTip" >
      <string/>
     </property>
     <property name="styleSheet" >
      <string notr="true" >background-color: rgb(216, 225, 255);</string>
     </property>
     <property name="horizontalScrollBarPolicy" >
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="autoScroll" >
      <bool>true</bool>
     </property>
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
     <property name="selectionMode" >
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior" >
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="showGrid" >
      <bool>true</bool>
     </property>
     <column>
      <property name="text" >
       <string>InvestigationId</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>RbNumber</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Title</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Type</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Instrument</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Investigator</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>RunRange</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Year</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Abstract</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Investigators Name</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Samples Name</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
  <widget class="QLineEdit" name="endRunEdit" >
   <property name="geometry" >
    <rect>
     <x>140</x>
     <y>68</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>End Run Number</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="startRunEdit" >
   <property name="geometry" >
    <rect>
     <x>59</x>
     <y>68</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>Start Run Number</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="casesensitiveBox" >
   <property name="geometry" >
    <rect>
     <x>530</x>
     <y>104</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>Case Sensitive keyWord Search</string>
   </property>
   <property name="text" >
    <string>Case Sensitive</string>
   </property>
  </widget>
  <widget class="QLabel" name="runNumberLabel" >
   <property name="geometry" >
    <rect>
     <x>14</x>
     <y>69</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Run#:</string>
   </property>
  </widget>
  <widget class="QLabel" name="instrumentLabel" >
   <property name="geometry" >
    <rect>
     <x>14</x>
     <y>104</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Instrument:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="instrmentBox" >
   <property name="geometry" >
    <rect>
     <x>140</x>
     <y>104</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>Instrument Box</string>
   </property>
  </widget>
  <widget class="QLabel" name="keywordsLabel" >
   <property name="geometry" >
    <rect>
     <x>450</x>
     <y>70</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>KeyWord(s):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="keywordslineEdit" >
   <property name="geometry" >
    <rect>
     <x>530</x>
     <y>70</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="enddateLineEdit" >
   <property name="geometry" >
    <rect>
     <x>291</x>
     <y>105</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>DD/MM/YYYY</string>
   </property>
  </widget>
  <widget class="QToolButton" name="startdatetoolButton" >
   <property name="geometry" >
    <rect>
     <x>400</x>
     <y>70</y>
     <width>25</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>...</string>
   </property>
  </widget>
  <widget class="QToolButton" name="enddatetoolButton" >
   <property name="geometry" >
    <rect>
     <x>400</x>
     <y>106</y>
     <width>25</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="startdateLineEdit" >
   <property name="geometry" >
    <rect>
     <x>291</x>
     <y>69</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>DD/MM/YYYY</string>
   </property>
  </widget>
  <widget class="QLabel" name="enddateLabel" >
   <property name="geometry" >
    <rect>
     <x>226</x>
     <y>105</y>
     <width>48</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>End Date:</string>
   </property>
  </widget>
  <widget class="QLabel" name="startdateLabel" >
   <property name="geometry" >
    <rect>
     <x>226</x>
     <y>69</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Start Date:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="searchButton" >
   <property name="geometry" >
    <rect>
     <x>689</x>
     <y>121</y>
     <width>70</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>Search Investigations</string>
   </property>
   <property name="text" >
    <string>Search</string>
   </property>
  </widget>
  <zorder>searchtreeWidget</zorder>
  <zorder>searchlabel</zorder>
  <zorder>endRunEdit</zorder>
  <zorder>startRunEdit</zorder>
  <zorder>casesensitiveBox</zorder>
  <zorder>runNumberLabel</zorder>
  <zorder>instrumentLabel</zorder>
  <zorder>instrmentBox</zorder>
  <zorder>keywordsLabel</zorder>
  <zorder>keywordslineEdit</zorder>
  <zorder>startdateLabel</zorder>
  <zorder>enddateLabel</zorder>
  <zorder>searchtableWidget</zorder>
  <zorder>startdateLineEdit</zorder>
  <zorder>enddatetoolButton</zorder>
  <zorder>startdatetoolButton</zorder>
  <zorder>enddateLineEdit</zorder>
  <zorder>searchButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
